src/app/zap-templates/zcl/data-model/chip/push-av-stream-transport-cluster.xml

```diff
@@ -0,0 +1,229 @@
+  </struct>
+
+  <cluster>
+    <domain name="Cameras"/>
+    <name>Push AV Stream Transport</name>
+    <code>0x0555</code>
+    <define>PUSH_AV_STREAM_TRANSPORT_CLUSTER</define>
+    <description>This cluster implements the upload of Audio and Video streams from the Camera AV Stream Management Cluster using suitable push-based transports.</description>
+    <client init="false" tick="false">true</client>
+    <features>
+      <feature bit="0" code="PERZONESENS" name="PerZoneSensitivity" summary="Supports a sensitivity value per Zone">
+        <optionalConform/>
+      </feature>
+    </features>
+    <server init="false" tick="false">true</server>
+    <globalAttribute code="0xFFFD" side="either" value="1"/>
+    <attribute code="0x0000" side="server" define="SUPPORTED_CONTAINER_FORMATS" type="SupportedContainerFormatsBitmap" min="0x00" max="0x01">SupportedContainerFormats</attribute>
+    <attribute code="0x0001" side="server" define="SUPPORTED_INGEST_METHODS" type="SupportedIngestMethodsBitmap" min="0x00" max="0x01">SupportedIngestMethods</attribute>
+    <attribute code="0x0002" side="server" define="CURRENT_CONNECTIONS" type="array" entryType="int16u">CurrentConnections</attribute>
+    <command code="0x00" source="client" name="AllocatePushTransport" optional="false" response="AllocatePushTransportResponse">
+      <description>This command SHALL allocate a transport and return a PushTransportConnectionID.</description>
+      <access op="invoke" privilege="manage"/>
+      <arg id="0" name="TransportOptions" type="TransportOptionsStruct"/>
+    </command>
+
+    <command code="0x01" source="server" name="AllocatePushTransportResponse" optional="false" disableDefaultResponse="true">
+      <description>This command SHALL be generated in response to an AllocatePushTransport command.</description>
+      <arg id="0" name="ConnectionID" type="int16u"/>
+      <arg id="1" name="TransportOptions" type="TransportOptionsStruct"/>
+      <arg id="2" name="TransportStatus" type="TransportStatusEnum" min="0x00" max="0x01"/>
+    </command>
+
+    <command code="0x02" source="client" name="DeallocatePushTransport" optional="false">
+      <description>This command SHALL be generated to request the Node deallocates the specified transport.</description>
+      <access op="invoke" privilege="manage"/>
+      <arg id="0" name="ConnectionID" type="int16u"/>
+    </command>
+
+    <command code="0x03" source="client" name="ModifyPushTransport" optional="false">
+      <description>This command is used to request the Node modifies the configuration of the specified push transport.</description>
+      <access op="invoke" privilege="manage"/>
+      <arg id="0" name="ConnectionID" type="int16u"/>
+      <arg id="1" name="TransportOptions" type="TransportOptionsStruct"/>
+    </command>
+
+    <command code="0x04" source="client" name="SetTransportStatus" optional="false">
```

Comment:
The spec has this bit (in the rendered text):

```
<<ref_PushTransport_MotionTriggerTimeControlStruct, Time Control Bounds]]
```
That's clearly broken and should be fixed.   Spec issue needed.

Reply:
fixed in the spec